packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.targets

```diff
@@ -155,4 +155,11 @@
   <!-- Import WPF Build logic only when we don't import NETFX's WinFX targets -->
   <Import Project="Microsoft.WinFX.targets" Condition="'$(ImportFrameworkWinFXTargets)' != 'true'"/>
 
+  <!--
+    Import Windows Forms targets.
+    These come via the Windows Forms transport package, that can be found under
+    https://github.com/dotnet/winforms/tree/main/pkg/Microsoft.Private.Winforms/sdk
+  -->
+  <Import Project="System.Windows.Forms.targets" />
```

Comment:
Always make sure the WinFX targets is imported at last.

Reply:
Is there a specific reason for this?

Reply:
Just a convention. I saw the content of the targets; it doesn't have much except a property to disable implicit references. For simple targets without any (_you know_) targets (_couldn't help myself_😂), placing it before any targets (_even the complex ones in WinFX_) is just easier to review, reason about and understand.

This is also why I moved the WinFX targets to bottom!

Reply:
These should only be imported if Condition="'$(UseWindowsForms)' == 'true'.

Reply:
Yes, we can conditionally import the targets
![image](https://user-images.githubusercontent.com/4403806/126591883-456284b7-5b68-4ace-b2a2-66371c111ee6.png)